README.md

```diff
@@ -55,6 +55,21 @@ yarn start
 
 The above command will run a web server on port :5173 that will serve the UI assets, with hot reloading support, and it will proxy all API requests to `http://localhost:16686` where Jaeger query should be running.
 
+------------
+### Using Windows OS for Development?
+
+While we don't natively support Windows OS for running the Jaeger UI Dev Environment, you can use WSL to run it.
+
+Here are some steps to follow:
+1. Install WSL: https://learn.microsoft.com/en-us/windows/wsl/install
+2. Install Node.JS: https://learn.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-wsl
```

Comment:
What about `yarn`? Any special instructions for it?

Reply:
It used to come pre-installed in Ubuntu 22 over WSL but I don't remember it exactly, so I have included a guide for it too.